app/src/components/WalletHistory/TxHistoryTable/TxHistoryTable.vue

```diff
@@ -9,7 +9,7 @@
       hide-default-footer
     >
       <template v-slot:default="props">
-        <TransactionDetails v-for="transaction in props.items" :key="transaction.id" :transaction="transaction" />
+        <TransactionDetails v-for="(transaction, index) in props.items" :key="index" :transaction="transaction" />
```

Comment:
change the key to be something other than the index

Reply:
Done